lib/service/AthleteApi.dart

```diff
@@ -37,4 +40,27 @@ class AthleteApi {
     print(athletesList);
     return athletesList;
   }
+
+  static Future<List<Athlete>> getAthletesFromIdsDict(
+      BuildContext context) async {
+    Map<String, List<int>> athleteIdDict = {
+      "ids": [9038, 22564, 21693, 18882],
+    };
+    final jsonRequestBody = json.encode(athleteIdDict);
+    print(jsonRequestBody);
+
+    final uri = Uri.parse('$_baseURL/nfl/players');
+    final athleteResponse = await http.post(uri,
+        headers: {"Content-Type": "application/json"}, body: jsonRequestBody);
+    print(athleteResponse.statusCode);
+
+    if (athleteResponse.statusCode == 200) {
+      var athleteResponseList = jsonDecode(athleteResponse.body) as List;
+      return athleteResponseList
+          .map((athlete) => Athlete.fromJson(athlete))
+          .toList();
+    } else {
+      throw Exception("Failed to load athlete");
```

Comment:
Does this just crash the app or it is handled somewhere else

Reply:
New commit now handles API errors in LandingPage by showing a "Reload Application" button.